packages/react-form-hooks/src/validation/predicates.ts

```diff
@@ -0,0 +1,35 @@
+export function lengthMoreThan(length: number) {
```

Comment:
These are exactly the same as the ones in `form-state`, and they aren't really specific to forms. I feel like we probably just want a `predicates` package in quilt that they both use. I'll make a PR for that too.